### DISCUSSION TITLE
Max AC shortcut

### DISCUSSION THREAD

**Discussion by sjetheredge** [Kemmey/Kemmey-TeslaWatch-Public#293] [Ideas]
Hi, love the app and use it all the time, there is one feature that I think would be super helpful. A Max AC shortcut is useful for those of us in hot climates. Right now we have to turn on climate and then adjust the temperature manually instead of a single shortcut to get the car cooled off as quick as possible. The use case is you are walking to your car that has been baking in the summer sun and you want to cool it off as quick as possible before you get in. 

Thanks!

**Comment by Kemmey** [comment]
Absolutely!

Yeah, I must admit that up until this winter, I figured turning on the hvac would be enough for the vehicle to figure it out - I mean, you request a certain cabin temperature, the vehicle should do it's best to achieve that. But obviously that's not what happens real world - if you want it hit NOW, you have to use defrost - of course the same applies to cold!

I'm finalizing 1.1.6 now, but I'll add that for 1.1.7. Thanks for bringing this to my attention! :-D

**Reply by calvarez** [reply]
I live in Phoenix, and my car lives outside in full sun.  I can measure a difference between putting it in max, or just letting the car figure it out.  So I stopped manually adjusting it.  I leave it on 70 and just let the car figure it.  This car has the best auto logic of any car I've owned, including much more expensive cars.

My shortcut sets HVAC on, stops the charger (for daytime cost reasons), and starts a 5 minute timer.  So I've measured the results repeatedly at the timer, nothing really changes.
